### DISCUSSION TITLE
I want to remove every other theme except two

### DISCUSSION THREAD

**Discussion by Ace-c** [prasanthrangan/hyprdots#1725] [General]
As per title say I wanna remove every other theme except Catppuccin-Mocha and decay green. I have old hyprdots config like 7 months ago. Can you tell which directory and what should I remove, and also what to change in bash script. And would I have to change something in gtk and qt5, qt6? if so what?

**Comment by kRHYME7** [comment]
* delete all hyprdots files.

* To delete themes navigate to ` ~/.config/hyde/themes/ ` then delete the directory representing the themes.

**Reply by Ace-c** [reply]
should I del all theme files according to this theme structure.. (It's bit dif from mine but I know) - ~
|⟼ /.config/
|    |
|    |⟼ hypr/themes/
|    |       ⮡ Synth-Wave.conf
|    |
|    |⟼ kitty/themes/
|    |       ⮡ Synth-Wave.conf
|    |
|    |⟼ Kvantum/Synth-Wave/
|    |       ⮡ Synth-Wave.kvconfig
|    |       ⮡ Synth-Wave.svg
|    |
|    |⟼ qt5ct/colors/
|    |       ⮡ Synth-Wave.conf
|    |
|    |⟼ qt6ct/colors/
|    |       ⮡ Synth-Wave.conf
|    |
|    |⟼ rofi/themes/
|    |       ⮡ Synth-Wave.rasi
|    |
|    |⟼ swww/
|    |       ⮡ Synth-Wave/*               # place wallpapers here
|    |
|    |⟼ waybar/themes/
|            ⮡ Synth-Wave.css
| 
|⟼ /.icons/
|       ⮡ <icon-pack>/                    # for icons
|       ⮡ <cursor-pack>/                  # for cursors
| 
|⟼ /.themes/
        ⮡ Synth-Wave/                     # main theme for GTK apps

**Reply by kRHYME7** [reply]
This structure is the old one right?. I think you can delete them all.

**Reply by Ace-c** [reply]
Yeah, any change to qt5 or 6? or leave it as it is

**Reply by kRHYME7** [reply]
> To delete themes navigate to ~/.config/hyde/themes/ then delete the directory representing the themes

this one is relevant for the newer structure where themes are all stored on a single Theme-directory. 

So in the newer structure,deleting theme is just simply deleting the theme folder.

For the older structure. I think just delete on the ./themes in the themes directory it is not needed anymore.

~Are you using the latest commits?~
Ops you're using the older one. You're on the right track.

**Reply by kRHYME7** [reply]
> Yeah, any change to qt5 or 6? or leave it as it is

configs in  that, were not maintained anymore. After kde moved to plasma 6, It is the same time that hyprdots is re structured.

**Reply by Ace-c** [reply]
Well I do removed every theme except one also remove some scripts and changed some scripts ...and lot of stuff...cleared a mess ..Thanks
